web/packages/teleport/src/lib/term/tty.ts

```diff
@@ -80,6 +81,34 @@ class Tty extends EventEmitterWebAuthnSender {
     this.send(JSON.stringify(data));
   }
 
+  sendFileTransferRequest(location: string, download: boolean, file?: File) {
```

Comment:
If we decide to stick with the boolean argument, then I'd make a keyword argument out of it. Or perhaps make `sendFileTransferRequest` private and expose separate `sendFileDownloadRequest` & `sendFileUploadRequest`.

This would improve readability since when reading `tty.sendFileTransferRequest(location, false, file)`, it's not clear what `false` does.